android/app/src/main/java/io/mosip/residentapp/RNSecureKeystoreModule.java

```diff
@@ -175,6 +189,38 @@ public void clearKeys() {
 
   @ReactMethod(isBlockingSynchronousMethod = true)
   public boolean hasBiometricsEnabled() {
-    return deviceCapability.hasBiometricsEnabled();
+    return deviceCapability.hasBiometricsEnabled(getCurrentActivity());
+  }
+
+  @ReactMethod
+  public void retrieveKey(String alias, Promise promise){
+     try{
+      promise.resolve(keystore.retrieveKey(alias));
+     }
+     catch(Exception e){
+      promise.reject(e.getMessage());
+     }
+     
+  }
+
+  @ReactMethod
+  public void storeGenericKey(String publicKey, String privateKey, String account){
+    keystore.storeGenericKey(publicKey,privateKey,account);
+  }
+
+  @ReactMethod
+  public void retrieveGenericKey(String account,Promise promise){
+    try {
+      List<String> keyList=keystore.retrieveGenericKey(account);
+      WritableArray writableArray = Arguments.createArray();
+      for (String key : keyList) {
+        writableArray.pushString(key);
+    }
+    promise.resolve(writableArray);
+    } catch (Exception e) {
+      promise.reject(e.getMessage());
+    }
+    
+   
```

Comment:
Add another functions to store and get key-value not related to private/public key but any key value

store(string key, string value)
get(string key, Promise promise)